test/extended/util/cli.go

```diff
@@ -35,10 +37,19 @@ type CLI struct {
 	finalArgs       []string
 	stdout          io.Writer
 	verbose         bool
+	execBashArgs    bool
 	cmd             *cobra.Command
 	kubeFramework   *e2e.Framework
 }
 
+type ExitError struct {
+	ExitStatus int
+}
+
+func (e ExitError) Error() string {
+	return fmt.Sprintf("Program exited with status %d", e.ExitStatus)
```

Comment:
why is this needed? You can just add exit code the the Verbose() output. (or to the default output when the error occurs)